addons/skin.estuary/xml/Home.xml

```diff
@@ -793,7 +793,7 @@
 						<param name="button2_onclick" value="EjectTray()"/>
 						<param name="button3_label" value="$LOCALIZE[600]"/>
 						<param name="button3_onclick" value="RipCD"/>
-						<param name="visible_3" value="String.IsEqual(System.DVDLabel,Audio-CD)"/>
+						<param name="visible_3" value="String.IsEqual(System.DVDLabel,Audio-CD) | String.IsEqual(System.DVDLabel,Audio CD)"/>
```

Comment:
```suggestion
						<param name="visible_3" value="String.Contains(System.DVDLabel,Audio) + String.Contains(System.DVDLabel,CD)"/>
```

Reply:
but that's less accurate, isn't it?
it would match 'NotAnAudioCD' for instance ;-)

Reply:
Hah, I guess so. How is the actual label filled? By the OS somehow?